### PR TITLE
Formatter: Update ecip-1083.md

### PR DESCRIPTION
**Pull request by realcodywburns**
add lang

### CODE REVIEW

File: _specs/ecip-1087.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 discussions-to: TBD
+author: meowsbits
 status: Withdrawn
-type: Standards Track (Core)
+category: Core
+type: Standards Track
 created: 2020-02-20
-license: Apache-2.0
-requires: ECIP1061 ECIP1078 ECIP1086
+license: Apache-2.0
```
Comment (BelfordZ) [inline _specs/ecip-1087.md:11]:
```suggestion
license: Apache-2.0
```

File: _specs/ecip-1087.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+author: meowsbits
 status: Withdrawn
-type: Standards Track (Core)
+category: Core
+type: Standards Track
 created: 2020-02-20
-license: Apache-2.0
-requires: ECIP1061 ECIP1078 ECIP1086
+license: Apache-2.0	
+requires: ECIP1061 ECIP1078 ECIP1086
```
Comment (BelfordZ) [inline _specs/ecip-1087.md:12]:
```suggestion
requires: ECIP1061 ECIP1078 ECIP1086
```

**Comment by realcodywburns** [comment]
build is failing with: jekyll 3.6.2 | Error:  uninitialized constant Faraday::Error::ConnectionFailed
possible solution https://github.com/github/personal-website/issues/166

**Comment by meowsbits** [comment]
@realcodywburns thanks for finding that fix. worked perfect.

**Pull request merged**